src/libraries/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj

```
@@ -63,6 +63,9 @@
     <!--Reference to System.Resources.Extensions is required in order to be able to embed a resource that requires a custom reader-->
     <ProjectReference Include="$(LibrariesProjectRoot)System.Resources.Extensions\src\System.Resources.Extensions.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <RuntimeHostConfigurationOption Include="System.Drawing.EnableUnixSupport" Value="true" />
+  </ItemGroup>
```

```suggestion
  <ItemGroup>
```
This is dead switch. It can be deleted.

---

Even better, thanks!